### PR TITLE
Show all results publiclly.

### PR DESCRIPTION
**Pull request by puqeko**
Previously students results would only be shown on the public
scoreboard if equal to or better than the median score. I don't see any point to this - we may as well show the preliminary results publicly so people can keep an eye on the scoreboard without logging in.

### CODE REVIEW

**Comment by puqeko** [comment]
2020-04-19 at 12 28 53 AM
2020-04-19 at 12 27 49 AM
2020-04-19 at 12 27 20 AM

File: app/views/contests/_index.html.erb [hunk 1]
```diff
@@ -27,8 +26,7 @@
     <td><%= contest.name %></td>
     <td><%= contest.start_time.strftime("%b %d, %H:%M") unless contest.start_time.nil? %></td>
     <td><%= contest.end_time.strftime("%b %d, %H:%M") unless contest.end_time.nil? %></td>
-    <td><%= contest.duration %></td>
-    <td><%= contest.owner_id %></td>
+    <td><%= contest.duration %> hours</td>
```
Comment (puqeko) [inline app/views/contests/_index.html.erb:29]: Self reminder to correctly pluralise this

File: app/views/contests/edit.html.erb [hunk 2]
```diff
@@ -3,13 +3,4 @@
...
-<% if false # COMMENTED %>
-<p>
-<b> Problems: </b>
-<% @problems.each do |problem| %>
-  <%= link_to problem.name, problem_path(problem) %>
-  (<%= link_to "remove", :controller => "problem_contest", :action => "remove", :method => "post", :problem_id => problem, :contest_id => @contest %>)
-<% end %>
-</p>
-<% end %>
+</p>
```
Comment (puqeko) [inline app/views/contests/edit.html.erb:6]: Newlines

File: app/views/contests/_admin.html.erb [hunk 3]
```diff
@@ -16,3 +16,7 @@
 <% end %>
 </ul>
 
+<p>
+  <b> Problem set: </b>
+  <%= link_to @contest.problem_set.name, @contest.problem_set %>
+</p>
```
Comment (puqeko) [inline app/views/contests/_admin.html.erb:22]: Newline

File: app/assets/stylesheets/main_table.css [hunk 4]
```diff
@@ -70,3 +70,6 @@ table.main_table > thead > tr > th {
   font-weight: bold;
 }
 
+.unofficial > td {
+  opacity: 0.4;
+}
```
Comment (puqeko) [inline app/assets/stylesheets/main_table.css:75]: Newline

File: db/migrate/20200418113601_add_offical_competitors.rb [hunk 5]
```diff
@@ -0,0 +1,5 @@
+class AddOfficalCompetitors < ActiveRecord::Migration
+    def change
+      add_column :contests, :show_unofficial_competitors, :boolean, :default => false
+    end
+  end
```
Comment (puqeko) [inline db/migrate/20200418113601_add_offical_competitors.rb:5]: Newline

File: db/migrate/20200418113600_add_live_scoreboard_field.rb [hunk 6]
```diff
@@ -0,0 +1,5 @@
+class AddLiveScoreboardField < ActiveRecord::Migration
+    def change
+      add_column :contests, :live_scoreboard, :boolean, :default => true
+    end
+  end
```
Comment (puqeko) [inline db/migrate/20200418113600_add_live_scoreboard_field.rb:5]: Newline

File: app/views/contests/_index.html.erb (hunk 1 above)
Comment (Holmes98) [reply app/views/contests/_index.html.erb:29]: I think this is fine actually, "1.0 hours" sounds more correct than "1.0 hour".
https://english.stackexchange.com/questions/44489/is-1-0-singular-or-plural

**Pull request closed without merging**